smith.lic

```diff
@@ -99,7 +109,19 @@ class Smith
 
   def stamp_item(recipe)
     fput('get my stamp')
-    fput("mark my #{recipe['noun']} with my stamp")
+    fput("mark  def check_oil(discipline)
+    $ORDINALS.each do |ord|
+	  case bput("look my #{ord} oil", /A thick and syrupy substance that easily coats and lubricates metal/, /You see nothing unusual/, /I could not find what you were referring to/, /There appears to be something written/, /(coal|lamp|fish) oil/, /The oil bottle is full/)
+	  when /A thick and syrupy substance that easily coats and lubricates metal/
+	    return if ord == 'first'
+	    bput("get my #{ord} oil", /You get/)
+		bput("stow oil", /You put/)
+		return
+	  when /I could not find what you were referring to/
+	    break
+	  end
+    end
+ my #{recipe['noun']} with my stamp")
```

Comment:
Something doesn't look right here. Copy paste into the middle of a line?

Reply:
Damn, it was fine on my end but when I pasted it, it looked wrong.  So I fixed it.  I guess it messed it up instead.